src/db.rs

```diff
@@ -1167,6 +1167,14 @@ impl DB {
         DBRawIterator::new_cf(self, cf_handle, &opts)
     }
 
+    pub fn raw_iterator_cf_opt(
```

Comment:
@rnarubin  I'd like to see some description for this **public** method

Reply:
@aleksuss you got it, pushed a change